### ISSUE TITLE
Add more logging to deployment

### ISSUE THREAD

**Issue by daltzctr** [wpilibsuite/deploy-utils#12]
Spawned from the following deployment failure that happens intermittent.

```
> Task :deployfrcCpproborio FAILED
    -F-> C:\Users\daltz\Documents\roboRIO Projects\testProject\build\exe\frcUserProgram\linuxathena\release\frcUserProgram -> frcUserProgram @ /home/lvuser

> Task :deployfrcStaticFileDeployroborio
    -C-> mkdir -p  @ /home/lvuser/deploy
      -[-1]
    5 file(s) are up-to-date and were not deployed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':deployfrcCpproborio'.
> A failure occurred while executing edu.wpi.first.deployutils.deploy.artifact.ArtifactDeployWorker
   > 4: Failure

* Try:
```

**Comment by ThadHouse** [comment]
Turns out this error is an internal sftp error, happening when copying a file. From what I can tell, it seems more likely to occur if the program is in a crash loop. I'm guessing a race condition inside of the SFTP library, which I plan on using a new one next season.

**Comment by ThadHouse** [comment]
This is going to be closed by #29. The logs ssh gives us should be better. That "Failure" is literally all the old setup gave us.
